.tests/jenkins/test_input_main.tsv

```diff
@@ -17,6 +17,7 @@ results/dna/HD827_T/additional_files/cnv/HD827_T.pathology.svdb_query.vcf	2c73b0
 results/dna/HD827_T/additional_files/cnv/HD827_T.purecn.amp_all_del_all.cnv_report.tsv	01660be1832c394b02e87e47e11afd9a
 results/dna/HD827_T/additional_files/cnv/HD827_T.purecn.amp_all_del_validated.cnv_report.tsv	83f8461e2ac1537ed67f5fbb48cc6e09
 results/dna/HD827_T/additional_files/cnv/HD827_T.purecn.svdb_query.vcf	9e1269f0072aa92eeaf969432af18ad8
+results/dna/HD827_T/additional_files/fusion/HD827_T.fuseq_wes.unfiltered.results.csv  XXX
```

Comment:
When will the checksums be updated?

Reply:
Either after run on Jenkins or manual run.

Reply:
Now the md5sums are updated